common/lib/xmodule/xmodule/video_module/video_handlers.py

```diff
@@ -166,6 +167,10 @@ def get_static_transcript(self, request, transcripts):
         if not self.transcript_language == 'en':
             return response
 
+        # If this video lives in library, the code below is not relevant and will error.
```

Comment:
The management command /edx-platform/cms/djangoapps/contentstore/management/commands/import.py calls import_course_from_xml. The only usages of import_library_from_xml at this time are the import handler called by Studio.

And given @bradenmacdonald's comment about no support for static assets in libraries, I think my proposed change to short-circuit get_static_transcript for libraries is fine. If I'm misunderstanding anyone, please let me know!